openedx_webhooks/templates/github_pr_cover_letter.md.j2

```diff
@@ -0,0 +1,47 @@
+{{ body }}
+
+## [PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)
+
+Thanks for the pull request, @{{ user }}!
+
+Please add a detailed description of your changes here.
+
+### Sandbox
+- [ ] Build a sandbox for your branch and add a link here
+
+### Testing
+- [ ] i18n
+- [ ] RTL
+- [ ] Unit, integration, acceptance tests as appropriate
+- [ ] Analytics
+- [ ] Performance
+- [ ] Database migrations are backwards-compatible
```

Comment:
Sounds good.  But should we wait until they've been finalized/moved to their proper destination?